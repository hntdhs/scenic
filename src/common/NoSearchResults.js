import React from "react";
import { NavLink } from "react-router-dom";

function backToSearch() {
    


return(
    <div>
        <h1>BIG PROBLEMS! No byways match what you searched for!</h1>
        <NavLink to={`/search`}>Try again! Try harder!</NavLink>
    </div>
)

};

export default backToSearch;