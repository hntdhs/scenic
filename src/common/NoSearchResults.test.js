import React from "react";
import { render } from "@testing-library/react";
import NoSearchResults from "./NoSearchResults.js"
import { BrowserRouter } from "react-router-dom";

it("matches snapshot", function () {
    const { asFragment } = render(<BrowserRouter><NoSearchResults /></BrowserRouter>);
    expect(asFragment()).toMatchSnapshot();
  });