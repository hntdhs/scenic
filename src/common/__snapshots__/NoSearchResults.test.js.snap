// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <h1>
      BIG PROBLEMS! No byways match what you searched for!
    </h1>
    <a
      href="/search"
    >
      Try again! Try harder!
    </a>
  </div>
</DocumentFragment>
`;
